### DISCUSSION TITLE
基于同步助手 1.1.0 标注嵌入模式 + Obsidian SimpRead Sync 2.1.0 文献笔记一站式解决方案

### DISCUSSION THREAD

**Discussion by 1wingedangel** [Kenshin/simpread#4084] [服务]
> 这是来自 [简悦社区](https://t.me/simpread) 用户 [1wingedangel](https://github.com/1wingedangel) 基于 [同步助手 1.1.0](https://github.com/Kenshin/simpread/discussions/4049#discussioncomment-2982511) 与 [标注嵌入模式](https://github.com/Kenshin/simpread/discussions/4070) 与 [SimpRead Unreader Sync 2.1.0](https://github.com/Kenshin/simpread/discussions/2889) 形成的新的工作流（模板）。

## 前置知识

 [同步助手 1.1.0](https://github.com/Kenshin/simpread/discussions/4049#discussioncomment-2982511) 与 [标注嵌入模式](https://github.com/Kenshin/simpread/discussions/4070) 与 [SimpRead Unreader Sync 2.1.0](https://github.com/Kenshin/simpread/discussions/2889) 实现：

集成标注到 Obsidian，在 Obsidian 中标注并实时同步标注到 Obsidian，不仅如此还可以管理你的稍后读。

## 视频演示

https://user-images.githubusercontent.com/81074/174554122-05fb0f52-803a-4131-949e-84421f2e8963.mp4

***

## 工作流

加入稍后读、阅读模式编辑稍后读元数据、live editor后生成快照，接下来就在 Obsidian 完成。

## 新模板

针对简悦新推出的极速版，分享一下 [新版 Obsidian Simpread Sync 插件的模板](https://github.com/Kenshin/simpread/discussions/2153#discussioncomment-2983977)

![image](https://user-images.githubusercontent.com/81074/174555638-2622b024-a09d-4924-a2a3-3b20e4af1f35.png)

## 关于简悦极速版结合 Hover Editor 的使用小结

https://user-images.githubusercontent.com/81074/174554065-ef5fa2ef-dcf2-4b31-8c13-324486afd1f8.mp4

简悦极速版结合 Hover Editor 可以实现稍后读的置顶浮动窗口，方便做一些更加复杂的工作流。

- 把任意文献笔记通过页面预览的方式创建 Hover Editor 窗口
- 激活 Hover Editor 的窗口聚焦（必须，除非你设置了自动聚焦）
- 打开触发简悦极速版的任意链接。此时会在 Hover Editor 的新面板里打开简悦极速版的稍后读页面
- 关闭原来的任意文献笔记

## 番外：用 Quicker 快速启动简悦极速版

https://github.com/Kenshin/simpread/discussions/4096

**Comment by Kenshin** [comment]
嗨，各位。

简悦官方针对新用户或不喜欢配置的用户，发布了一个新功能：**Obsidan 配置库**。

细节请看下面，几分钟就能配置成完成。

***

![img](https://imgs.zhubai.love/c626678ff6b64dd1ac9dfdae6cf6d188.png)

> 针对新用户的一步到位的 Obsdian 配置库。

## 🗂 配置库

[简悦 · 配置库](https://www.yuque.com/kenshin/simpread/ds8zk0) 是简悦官方推出的一套针对新用户的极简配置方案，方便新用户用最快的方式使用简悦的各种高级服务，配置库内置了常用的双链笔记用法，如：Notion、Obsidian、Logseq、Roam Research，同时包含了简悦在阅读模式上的一些常规插件：Live Editor、题图、Safari 阅读模式等。

## 💡 特点

用户仅需要下载对应的配置库，修改里面的 UID 为自己的高级账户 UID，即可实现简悦的各种高级操作。

## 💁‍♂️ 受众

新用户或者不想使用手动配置的用户。

## 📝 如何使用

1️⃣ 下载

2️⃣ 修改为自己的 UID

3️⃣ 将必要的文件夹复制粘贴到相应位置

✅ Done！

## 📚 Obsidian 配置库

## 📝 特点

1️⃣ 加入稍后读，自动将本地快照和 Markdown 导入到 Obsidian

2️⃣ 加入标注时，自动将标注内容导入本地快照和 Obsidian

3️⃣ 改变稍后读或标注的元数据时，自动将改动的内容保存到本地快照和 Obsidian

4️⃣ 在 Obsidian 中直接内置好了简悦的标注系统，可直接在 Obsidian 实现标注

## 📖 区别

![img](https://imgs.zhubai.love/f4a51d8e8fb4401281b07f82bf0bc7d1.png)

## 📘 极速版

🔗 [Github](https://github.com/Kenshin/simpread-configs/blob/main/obsidian@little/Getting%20Started.md) | [语雀](https://www.yuque.com/kenshin/simpread/xkuecp)

不需要同步助手，无任何配置，支持 Markdown 形式的快照，自动同步标注，适合 Obsidain + 简悦的尝鲜用户。

## 📙 基础版

🔗 [Github](https://github.com/Kenshin/simpread-configs/blob/main/obsidian@localrestapi/Getting%20Started.md) | [语雀](https://www.yuque.com/kenshin/simpread/cg33gh)

需要同步助手，几乎无任何配置，支持本地快照，自动同步标注，适合 Obsidain + 简悦的轻量级用户。

## 📗 高级版

🔗 [Github](https://github.com/Kenshin/simpread-configs/blob/main/obsidian@sync/Getting%20Started.md) | [语雀](https://www.yuque.com/kenshin/simpread/wq35mh)

需要同步助手，几乎无任何配置，支持本地快照，自动同步标注，可在 Obsidian 中标注，适合 Obsidain + 简悦的重度用户。

**Comment by EdisonWendy** [comment]
请问这个标注框的颜色随着标注文字的颜色变化的样式怎么设置，我将标注模板中的代码作为CSS代码片段导入外观也没有文中截图这种效果
![微信截图_20230311135455](https://user-images.githubusercontent.com/120798970/224467706-1308132c-65e7-4c53-aac1-8d2f8bbad3c9.png)
